src/widgets/radio.jsx

```diff
@@ -344,6 +357,13 @@ var BaseRadio = React.createClass({
         this.props.onCheckedChange(newChecked);
     },
 
+    noneOfTheAboveIndex: function() {
+        choices = _.map(this.props.choices, (choice) => {
```

Comment:
Is there a reason why you're using a `map` instead of a `filter` for this? (Actually, even better might be `_.findIndex(this.props.choices, 'isNoneOfTheAbove')` to save you from the `indexOf` immediately after)